### ISSUE TITLE
ArgumentException: An item with the same key has already been added

### ISSUE THREAD

**Issue by Moj-FPS** [shudnal/ProtectiveWards#2]
Hi there. While I was setting up the ward, I noticed that none of the cosmetic options worked at all- no bubble, no color changes, etc. Everything else seems to work okay (haven't attempted to taxi anywhere yet) but the repairs, flashes, damage modifiers, etc. all work fine. When I look at the ward, I repeatedly receive this error in the console:

```
[Error  : Unity Log] ArgumentException: An item with the same key has already been added. Key: (-5.56, 50.69, 4.66)
Stack trace:
System.Collections.Generic.Dictionary`2[TKey,TValue].TryInsert (TKey key, TValue value, System.Collections.Generic.InsertionBehavior behavior) (at <17d9ce77f27a4bd2afb5ba32c9bea976>:0)
System.Collections.Generic.Dictionary`2[TKey,TValue].Add (TKey key, TValue value) (at <17d9ce77f27a4bd2afb5ba32c9bea976>:0)
ZoneSystem.GetLocationIcons (System.Collections.Generic.Dictionary`2[TKey,TValue] icons) (at <1f17a254067e41d0b59746dec98ad607>:0)
ProtectiveWards.ProtectiveWards+PrivateArea_AddUserList_WardAltActionCaption.Prefix (PrivateArea __instance, System.Text.StringBuilder text) (at <f16c1da8ec824ab5a92b4c16eeedeefe>:0)
(wrapper dynamic-method) PrivateArea.DMD<PrivateArea::AddUserList>(PrivateArea,System.Text.StringBuilder)
PrivateArea.GetHoverText () (at <1f17a254067e41d0b59746dec98ad607>:0)
Hud.UpdateCrosshair (Player player, System.Single bowDrawPercentage) (at <1f17a254067e41d0b59746dec98ad607>:0)
(wrapper dynamic-method) Hud.DMD<Hud::Update>(Hud)
```

I'm not sure if these two issues are related; I do have quite a few mods currently, but I'm not sure what could be causing them. Let me know if you need more information.

Thank you!

**Comment by Moj-FPS** [comment]
Update: 

The cosmetics are working now, but still getting the log errors when looking at the ward.

**Comment by shudnal** [comment]
Thanks. Fixed in 1.2.1.
